src/stores/currentUser.js

```diff
@@ -42,44 +44,44 @@ export default (connectors) => {
 
     async  sendForgotPassword (email) {
         try {
-          const status = await connectors.forgotPassword.send({email:email})
-          return status
+          await connectors.forgotPassword.send({email:email})
```

Comment:
In many cases I can see, that your code is not formatted by the standardjs formatting rules. May I ask you to run lint fix on your code before pushing it?